modules/java/pages/_partials/gs-additional-steps-for-linux.adoc

```diff
@@ -14,13 +14,15 @@ NOTE: These libraries are provided only in the `.zip` distributable.
 Web Service/Tomcat::
 +
 --
-. Copy the _native libraries_ (`<pathToCbl>/support`) to a location accessible to -- and usable by -- your build and runtime environments.
-. Add the following libraries to the `LD_LIBRARY_PATH` in `$CATALINA_BASE/bin/setenv.sh:
+. Within the `.zip` file, locate the support files appropriate for your environment.
+. Copy the directory `<exploded-distribution-zip>/support` to a location accessible and executable by your build and runtime environments.
 +
-NOTE: If the setenv.sh file doesn’t exist, you will need to create a new file.
-
+include::partial$directory-diagrams.adoc[tag=couchbase-lite-zip]
 +
-[source, bashrc, subs="normal"]
+. Add the path of the directories `libc++` and `libicu` to  `LD_LIBRARY_PATH` in `$CATALINA_BASE/bin/setenv.sh.
```

Comment:
plural?  There are two paths....

both here and in the next section

Reply:
Fixed